### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by duncanjbrown**
Revise and reorganise README to remove outdated info, standardise style, and slightly condense setup steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,39 +2,40 @@
...
+Once it is set up, every web request and database update, as permitted by
+configuration, will flow to BigQuery.
 
 It also provides a Rake task for backfilling BigQuery with entities created
-before you started sending events (see **Importing existing data** below), and
-another for keeping your field configuration up to date.
+before you started sending events.
 
-To set the gem up follow the steps in "Configuration", below.
+To set the gem up follow the steps in "Installation", below.
```
Comment (asatwal) [inline README.md:20]:
```suggestion
To set the gem up follow the steps in [Installation](#installation), below.
```

File: README.md [hunk 2]
```diff
@@ -324,25 +372,6 @@ The different types of events that DfE Analytics send are:
...
-
-```bash
-bundle exec rails dfe:analytics:import_entity[entity_name]
-```
-
-**IMPORTANT**:
-
-Do not run an import when there is a lot of traffic on the website. Consider running the import out of hours or when the traffic is minimal.
-
 ## Event debugging
```
Comment (asatwal) [inline README.md:375]:
```suggestion

## Event debugging
```

**Review by asatwal — APPROVED** [review]
LGTM - Just a few minor suggestions.

**Comment by duncanjbrown** [comment]
Changes accepted and fixed up, will merge on passing

**Pull request merged**